### ISSUE TITLE
Incorrect type in @types/dbus package

### ISSUE THREAD

**Issue by benfrancis** [DefinitelyTyped/DefinitelyTyped#71006]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/286e356cfaf3ad52bcd2da1e9bfc79f16b664a9b/types/dbus/index.d.ts#L30C71-L30C77

getProperty(name: string, callback: (err: Error | null, ~~name: string~~***value: any***) => void): void;

**Comment by mithelan** [comment]
Do we need to update the parameters from name to value?

**Comment by benfrancis** [comment]
Yes, I think the confusion probably comes from the [README](https://github.com/Shouqun/node-dbus/blob/master/README.md#interfaceprototypegetpropertypropertyname-callback) which lists `(err, name)` as the arguments to the callback, but the prose of the text correctly states that it actually returns the value, not the name of the property (which is hopefully obvious from the name of the function).
